python3Packages.catkin-pkg: 1.0.0 -> 1.1.0

r-ryantm opened the pull request:
Automatic update generated by [nixpkgs-update](https://github.com/ryantm/nixpkgs-update) tools. This update was made based on information from passthru.updateScript.


meta.description for python3Packages.catkin-pkg is: Library for retrieving information about catkin packages

meta.homepage for python3Packages.catkin-pkg is: http://wiki.ros.org/catkin_pkg

meta.changelog for python3Packages.catkin-pkg is: https://github.com/ros-infrastructure/catkin_pkg/blob/1.1.0/CHANGELOG.rst


###### Updates performed
- Ran passthru.UpdateScript

###### To inspect upstream changes



- [Compare changes on GitHub](https://github.com/ros-infrastructure/catkin_pkg/compare/1.0.0...1.1.0)

###### Impact

<b>Checks done</b>

---

- built on NixOS
- The tests defined in `passthru.tests`, if any, passed
- found 1.1.0 with grep in /nix/store/1m0qplx8nk6j7j6drrgi5mdjzbqsplaw-python3.13-catkin-pkg-1.1.0
- found 1.1.0 in filename of file in /nix/store/1m0qplx8nk6j7j6drrgi5mdjzbqsplaw-python3.13-catkin-pkg-1.1.0

---

<details>
<summary>
<b>Rebuild report</b> (if merged into master) (click to expand)
</summary>

```
4 total rebuild path(s)

4 package rebuild(s)

First fifty rebuilds by attrpath

python312Packages.catkin-pkg
python313Packages.catkin-pkg
snapcraft
```

</details>

<details>
<summary>
<b>Instructions to test this update</b> (click to expand)
</summary>

---

Either **download from the cache**:
```
nix-store -r /nix/store/1m0qplx8nk6j7j6drrgi5mdjzbqsplaw-python3.13-catkin-pkg-1.1.0 \
  --option binary-caches 'https://cache.nixos.org/ https://nixpkgs-update-cache.nix-community.org/' \
  --option trusted-public-keys '
  nixpkgs-update-cache.nix-community.org-1:U8d6wiQecHUPJFSqHN9GSSmNkmdiFW7GW7WNAnHW0SM=
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  '
```
(The nixpkgs-update cache is only trusted for this store-path realization.)
For the cached download to work, your user must be in the `trusted-users` list or you can use `sudo` since root is effectively trusted.

Or, **build yourself**:
```
nix-build -A python3Packages.catkin-pkg https://github.com/r-ryantm/nixpkgs/archive/9593aa7609eebcf687f64782ef0d576ad5a54aa6.tar.gz
```
Or:
```
nix build github:r-ryantm/nixpkgs/9593aa7609eebcf687f64782ef0d576ad5a54aa6#python3Packages.catkin-pkg
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/1m0qplx8nk6j7j6drrgi5mdjzbqsplaw-python3.13-catkin-pkg-1.1.0
ls -la /nix/store/1m0qplx8nk6j7j6drrgi5mdjzbqsplaw-python3.13-catkin-pkg-1.1.0/bin
```

---

</details>
<br/>



### Pre-merge build results

We have automatically built all packages that will get rebuilt due to
this change.

This gives evidence on whether the upgrade will break dependent packages.
Note sometimes packages show up as _failed to build_ independent of the
change, simply because they are already broken on the target branch.

## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review`
Commit: `9593aa7609eebcf687f64782ef0d576ad5a54aa6`

---
### `x86_64-linux`
<details>
  <summary>:x: 2 packages failed to build:</summary>
  <ul>
    <li>snapcraft</li>
    <li>snapcraft.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 4 packages built:</summary>
  <ul>
    <li>python312Packages.catkin-pkg</li>
    <li>python312Packages.catkin-pkg.dist</li>
    <li>python313Packages.catkin-pkg</li>
    <li>python313Packages.catkin-pkg.dist</li>
  </ul>
</details>

---

###### Maintainer pings

cc @adhityaravi @bepri @dstathis for [testing](https://github.com/ryantm/nixpkgs-update/blob/main/doc/nixpkgs-maintainer-faq.md#r-ryantm-opened-a-pr-for-my-package-what-do-i-do).

> [!TIP]
> As a maintainer, if your package is located under `pkgs/by-name/*`, you can comment **`@NixOS/nixpkgs-merge-bot merge`** to automatically merge this update using the [`nixpkgs-merge-bot`](https://github.com/NixOS/nixpkgs-merge-bot).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

r-ryantm committed:
python3Packages.catkin-pkg: 1.0.0 -> 1.1.0

jnsgruk committed:
snapcraft: add `catkin-pkg` to `pythonRelaxDeps`

jnsgruk commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 443606`
Commit: `eaa19d3ea883ffe9e309745f4896de029cef01a2`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 6 packages built:</summary>
  <ul>
    <li>python312Packages.catkin-pkg</li>
    <li>python312Packages.catkin-pkg.dist</li>
    <li>python313Packages.catkin-pkg</li>
    <li>python313Packages.catkin-pkg.dist</li>
    <li>snapcraft</li>
    <li>snapcraft.dist</li>
  </ul>
</details>

jnsgruk committed:
snapcraft: switch LXD socket path to Incus socket path

LXD has been removed from NixOS unstable due
to lack of maintenance, but the Incus socket
should be a drop in replacement.

jnsgruk committed:
python3Packages.craft-providers: use Incus socket

LXD has been removed from nixpkgs unstable due to
lack of maintenance, but the Inxus socket should
be a drop-in replacement.

This commit switches the socket path to the Incus
unix socket.

jnsgruk commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 443606`
Commit: `6562c2a9d4fcb89857b7965652d170a6694b020f`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 18 packages built:</summary>
  <ul>
    <li>charmcraft</li>
    <li>charmcraft.dist</li>
    <li>python312Packages.catkin-pkg</li>
    <li>python312Packages.catkin-pkg.dist</li>
    <li>python312Packages.craft-application</li>
    <li>python312Packages.craft-application.dist</li>
    <li>python312Packages.craft-providers</li>
    <li>python312Packages.craft-providers.dist</li>
    <li>python313Packages.catkin-pkg</li>
    <li>python313Packages.catkin-pkg.dist</li>
    <li>python313Packages.craft-application</li>
    <li>python313Packages.craft-application.dist</li>
    <li>python313Packages.craft-providers</li>
    <li>python313Packages.craft-providers.dist</li>
    <li>rockcraft</li>
    <li>rockcraft.dist</li>
    <li>snapcraft</li>
    <li>snapcraft.dist</li>
  </ul>
</details>

jnsgruk commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 443606`
Commit: `d23d44d121753143451128011ce5f26e67f1e01b`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 18 packages built:</summary>
  <ul>
    <li>charmcraft</li>
    <li>charmcraft.dist</li>
    <li>python312Packages.catkin-pkg</li>
    <li>python312Packages.catkin-pkg.dist</li>
    <li>python312Packages.craft-application</li>
    <li>python312Packages.craft-application.dist</li>
    <li>python312Packages.craft-providers</li>
    <li>python312Packages.craft-providers.dist</li>
    <li>python313Packages.catkin-pkg</li>
    <li>python313Packages.catkin-pkg.dist</li>
    <li>python313Packages.craft-application</li>
    <li>python313Packages.craft-application.dist</li>
    <li>python313Packages.craft-providers</li>
    <li>python313Packages.craft-providers.dist</li>
    <li>rockcraft</li>
    <li>rockcraft.dist</li>
    <li>snapcraft</li>
    <li>snapcraft.dist</li>
  </ul>
</details>
